{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -5,29 +5,109 @@
 #################################################################################
 
 BUCKET = {{ cookiecutter.s3_bucket }}
+PROJECT_NAME = {{ cookiecutter.repo_name }}
+PYTHON_INTERPRETER = {{ cookiecutter.python_interpreter }}
```

Comment:
Yeah, that seems right but there's an added wrinkle for projects that don't use virtualenv:
we ask for the Python interpreter during setup without suggesting (or requiring) that that interpreter be used to setup a virtualenv. In that case, you might expect that setting the Python interpreter be used for any commands where Python gets called.

So, I think there are two options that would work:
 1) `[OPTIONAL] Choose Python interpreter for virtualenv`
 2) Use the interpreter passed in setup for Make commands. I haven't tested this, but I assume that if I do `--python=python3` when I create my virtualenv, calling `python3` at the command line will still get the right one when I have that env active.

I'm leaning towards 2 since it covers both use cases. Thoughts?